daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -234,6 +234,9 @@ enum PrimType {
   // *Available in versions >= 1.dev*
   NUMERIC = 17;
 
+  // Builtin type 'Any'
+  // *Available in versions >= 1.dev*
+  ANY = 18;
```

Comment:
Scala people would prefer not using `Any`. (cc @gerolf-da)

Reply:
So what are you proposing instead? Calling this `Dynamic` while keeping `AnyTemplate` seems really confusing and I’m not entirely sold that `DynamicTemplate` is a good name.

Reply:
I’ll go for `Dynamic` now and keep `AnyTemplate`. I’ll rename it in a separate PR to not clutter the diff here too much.

Reply:
OK let's go with `ANY`.